src/frontend/classic/components/MenuLeftPanel.svelte

```diff
@@ -167,6 +172,36 @@
                     items={themes}
                 />
             </SettingItem>
+            <SettingItem
+                labelText="Viewer mode"
+                helperText="You can change the viewer mode (webtoon or manga)"
+            >
+                <SettingSelect
+                    store={viewerMode}
+                    storageKey={settings.VIEWER_MODE.KEY}
+                    items={viewerModesSelect}
+                />
+            </SettingItem>
+            {#if $viewerMode === "manga"}
+                <SettingItem
+                    labelText="Inverse reading"
+                    helperText="Inverse the reading (like a real manga)"
```

Comment:
Just "Inverse the reading direction"

"like a real manga" might not be the reason